org.jacoco.tests/pom.xml

```diff
@@ -33,6 +33,7 @@
     <module>../org.jacoco.ant.test</module>
     <module>../jacoco-maven-plugin.test</module>
     <module>../org.jacoco.examples.test</module>
+    <module>../org.jacoco.tests.coverage</module>
```

Comment:
Actually I also tried this: I had trouble with dependencies to same module with different qualifiers (it makes appearing those bundles twice in the report). Nut sure whether we actually need these dependencies as we now create the download zip in a separate project.